.gitignore

```diff
@@ -31,3 +31,6 @@
 /config/master.key
 coverage/
 .env
+
+# Ignore DS Store
+**/.DS_Store
```

Comment:
**suggestion:**: It's fine to have this here, but you could also ad `.DS_Store` to your global `.gitignore` (in your home folder).

That said, I think `.DS_Store` instead of `**/.DS_Store` would be enough.